src/main/java/edu/ucla/library/bucketeer/Item.java

```diff
@@ -98,8 +95,8 @@ public IFilePathPrefix getFilePathPrefix() {
      */
     @JsonIgnore
     public WorkflowState getWorkflowState() throws IllegalArgumentException {
-        return StringUtils.trimToNull(myWorkflowState) == null ? WorkflowState.EMPTY : WorkflowState.valueOf(
-                myWorkflowState.toUpperCase(Locale.US));
+        return StringUtils.trimToNull(myWorkflowState) == null ? WorkflowState.EMPTY
+                : WorkflowState.valueOf(myWorkflowState.toUpperCase(Locale.US));
```

Comment:
I did used to have my line length at 118 but changed to 120 awhile back. I believe that to be the line lengthening issue.

Reply:
Question about the "The README also needs info about creating /etc/bucketeer/bucketeer-features.conf and installing Kakadu in order to run locally."

I'm not sure we want people to run this locally vs. within the container(?) I proposed it for testing the large image functionality because we're not currently using Git LFS but I really think it would be better to devise a way to share the large image tiff and let people run this functionality from within the ff-on docker container. If we could move testing of this feature into the ff-on container do you still think we'd want people to run these particular tests from the host machine?

Reply:
Oh okay, good to know re: formatting. I knew you'd updated the IDE itself but I hadn't considered that maybe Eclipse changed their code formatting algorithm.

Reply:
It still would be good if we could get to the point where stray formatting changes didn't show up. I keep hoping a set of rules can be shared and used, but I guess I could also fall back to just having it check lines I've changed :-(

Reply:
Now that you mention the 118 -> 120 change I think that may account for a lot of them. I feel like I haven't seen too many ternaries.

Reply:
With the README, I assumed we wanted to leave running via `mvn -Plive test` open as an option since we have the Running the Application for Development section. My personal inclination would be to run it that way for development (assuming it would be easier to setup a debugger vs. when running inside of a container).

Reply:
Yes, I think that's a safe assumption. Okay, will update README.